src/pages/solutions.astro

```diff
@@ -90,7 +90,12 @@ const page: IStandardPageMetadata = {
       alt="a satellite in orbit around the earth with a GPS signal coming from it"
       class="w-full max-w-[400px] rounded-2xl"
     />
-    <p>text</p>
+    <p>
+      We're proud to offer surveyors centimeter-level precision measurements through our TURN GPS Network. The TURN
+      Network is a high-precision global navigation satellite system (GNSS) providing real-time corrections and data for
+      post processing. Surveyors in Utah, Nevada, Idaho, and Wyoming are taking advantage of this useful service for a
+      variety of purposes including research, engineering, drone operations, and more!
+    </p>
```

Comment:
There are more things to offer surveyors than TURN 

<img width="139" alt="image" src="https://github.com/agrc/gis.utah.gov/assets/325813/a22bc000-5a23-4bdb-bb09-53f36f3595b0">